clientv3/integration/network_partition_test.go

```diff
@@ -70,6 +70,14 @@ func testBalancerUnderNetworkPartition(t *testing.T, op func(*clientv3.Client, c
 	}
 	defer cli.Close()
 
+	// wait for ep[0] to be pinned
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
```

Comment:
we used this in a lot of places. can you abstract this to a func like waitPinReady or something?